src/main/scala/uk/gov/nationalarchives/fileformat/FileUtils.scala

```diff
@@ -10,6 +10,9 @@ import com.typesafe.config.ConfigFactory
 import graphql.codegen.GetOriginalPath.getOriginalPath.{Data, Variables, document}
 import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.GetObjectRequest
+import sttp.client.{HttpURLConnectionBackend, Identity, NothingT, Response, SttpBackend}
+import sttp.client.asynchttpclient.WebSocketHandler
+import sttp.client.asynchttpclient.future.AsyncHttpClientFutureBackend
```

Comment:
Think `sttp.client.asynchttpclient.future.AsyncHttpClientFutureBackend` is an unnecessary import